Blockchain API migration information

samgutentag opened the pull request:
Sourced data from [spreadsheet](https://docs.google.com/spreadsheets/d/1q2tt58VSl7MCRbZKHDwC27bd8aKtVOk9ntdvHyQOxdE/edit#gid=0)

- move blockchain api sidebar under architecture category
- sort pages alphabetically
- each page has sections for "available on Oracles API, available on Solana API, and Deprecated" where relevant
- updated warning admonition text

samgutentag committed:
‘accounts’ endpoint updates

samgutentag committed:
‘accounts’ endpoint updates

samgutentag committed:
‘assert locations’ endpoint updates

samgutentag committed:
‘assert locations’ endpoint updates

samgutentag committed:
‘blocks’ endpoint updates

samgutentag committed:
‘blocks’ endpoint updates

samgutentag committed:
‘chiain vars’ endpoint updats

samgutentag committed:
‘chiain vars’ endpoint updats

samgutentag committed:
‘challenges’ endpoint updates

samgutentag committed:
‘challenges’ endpoint updates

samgutentag committed:
‘cities’ endpoint updates

samgutentag committed:
‘cities’ endpoint updates

samgutentag committed:
‘dc burns’ endpoint updates

samgutentag committed:
‘dc burns’ endpoint updates

samgutentag committed:
‘elections’ endpoint updates

samgutentag committed:
‘elections’ endpoint updates

samgutentag committed:
show table of contents on ‘elections’ page

samgutentag committed:
show table of contents on ‘elections’ page

samgutentag committed:
‘hotspots’ endpoint updates

samgutentag committed:
‘hotspots’ endpoint updates

samgutentag committed:
‘location’ endpoint updates

samgutentag committed:
‘location’ endpoint updates

samgutentag committed:
‘oracle prices’ endpoint updates

samgutentag committed:
‘oracle prices’ endpoint updates

samgutentag committed:
‘ouis’ endpoint updates

samgutentag committed:
‘ouis’ endpoint updates

samgutentag committed:
‘pending transactions’ endpoint updates

samgutentag committed:
‘pending transactions’ endpoint updates

samgutentag committed:
‘rewards’ endpoint updates

samgutentag committed:
‘rewards’ endpoint updates

samgutentag committed:
‘state channels’ endpoint updates

samgutentag committed:
‘state channels’ endpoint updates

samgutentag committed:
‘stats’ endpoint updates

samgutentag committed:
‘stats’ endpoint updates

samgutentag committed:
‘transactions’ endpoint updates

samgutentag committed:
‘transactions’ endpoint updates

samgutentag committed:
‘validators’ endpoint updtes

samgutentag committed:
‘validators’ endpoint updtes

samgutentag committed:
update language for deprecated endpoint headers

samgutentag committed:
update language for deprecated endpoint headers

samgutentag committed:
sort side alphabetically

samgutentag committed:
sort side alphabetically

samgutentag committed:
updated disclaimer text

samgutentag committed:
updated disclaimer text

samgutentag committed:
fix link

samgutentag committed:
fix link

samgutentag committed:
update sidebars to put API under architecture

samgutentag committed:
update sidebars to put API under architecture

jthiller commented:
overall seems fine, but I don't think we should add out-of-date data into our nav unless absolutely necessary. Would recommend keeping it siloed in separate nav object.

btw rebased to bring this to HEAD

DanielChrobak committed:
Rewording (#1198)

I think we’ve been trying to use the word “reward” instead of “pay”

ChewingGlass committed:
Add better error messages (#1201)

Siegfried-B committed:
Update validators-and-voting-power.mdx (#1193)

* Update validators-and-voting-power.mdx

Correcting errors and suggesting improvements regarding: (a) HNT-veHNT positions, (b) DAO Utility Score

* line wrap formatting

* feedback

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>

ChewingGlass committed:
Fix solana signer block (#1202)

ChewingGlass committed:
Add instructions to sideload the ledger app (#1200)

* Add instructions to sideload the ledger app

* removing redirect

* Update docs/solana/migration/ledger.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* formatting fixes

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>
Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

waveform06 committed:
Update rewardable-entities.mdx (#1199)

ECC section spotted by user 👽 from Mars#5523 and notified on discord

samgutentag committed:
migration guide

samgutentag committed:
minor copy edits

DanielChrobak committed:
Grammar/Wording/Spelling Fixes (#1204)

* Grammar/Wording Change

* Grammar/Wording Change

* Grammar Change

* Grammar/Wording Changes

* Spelling Error Fix

* Fixing Bad Fix

* applying line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update network-user.mdx (#1205)

* Update network-user.mdx

Added post from lytz about no HNT to DC burn in console during migration

* line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

pankajrathore9599 committed:
Updated rewardable-entites.mdx file (#1189)

We don't have any function with `send` in the anchor, We have `sendAndConfirm` for executing the successful transaction in the anchor.

waveform06 committed:
Update mobile.mdx (#1206)

* Update mobile.mdx

Updated emissions schedule and references to other HIPs

* moving * to footnotes in table, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update mobile.mdx (#1207)

* Update mobile.mdx

Added Year 1 starts 1st August 2022.

* line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update iot.mdx (#1208)

* Update iot.mdx

Updated HIP references and added a Year 1 date

* moving asterisks to footnotes

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

deasydoesit committed:
Updating oracle data page in anticipation of new data on 4/18 (#1209)

* Updating oracle data page in anticipation of new data on 4/18

* Updating based on pre PR review

ChewingGlass committed:
Fix tx ordering in ledger migration (#1210)

samgutentag committed:
language updates

ChewingGlass committed:
Use correct uri (#1211)

* Fix tx ordering in ledger migration

* -Fix url

* Use devnet

* add https

ChewingGlass committed:
Add video walkthough. (#1212)

* Add video walkthough.

* extra line

* adding admonition box

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Stubs out veHNT doc.

Should break this out into 3 docs eventually.

jthiller committed:
Update veHNT

jthiller committed:
Adding more of a narrative to migration...

...moving veHNT content to dedicated page.

jthiller committed:
Copy tweaks

ChewingGlass committed:
Remove channels instructions (#1214)

* Remove channels instructions

* updated youtube walkthrough link

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

Talitrus committed:
Helium-Solana Ledger app clarifications (#1215)

* reorder steps to install Helium-Solana app

Developer mode must be enabled first in order to show the Helium-Solana app

* Remove mention of GitHub for Helium-Solana app

* add Helium-Solana Ledger devnet warning

* Remove ref to uninstalling Solana Ledger app

* formatting adjustments

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Maker Wallet SOL and DC (#1216)

* maker wallet SOL and DC needs

* nicer images and links to compression calculator

* typo

* feedback from review

* mint calculation tables

samgutentag committed:
link fixup (#1217)

samgutentag committed:
Update validator-operator.mdx (#1220)

Talitrus committed:
change 7d landrush to 10

samgutentag committed:
Hotspot App Version 5.0.0 docs updates (#1218)

* updated images

* Hotspot app docs redux

* formatting to maker apps page

* migration guide updates for new Hotspot app

jthiller committed:
Clean up Overview page (#1222)

* Update clock

* Clean up overview page. Add UTC time.

* More clock updates.

waveform06 committed:
Update validator-operator.mdx (#1223)

replaced some lost words.

samgutentag committed:
Helium Lorawan Roaming Start Guide (#1197)

* initial commit of Roaming quick start guide docs

* image updates and typo correction

Siegfried-B committed:
Update vehnt.mdx (#1224)

- Correcting lower limit for lock-up duration from 6 months to 1 day

jthiller committed:
Realms/Staking Docs (#1225)

* Staking with Realms
* Updates & linting

Siegfried-B committed:
Update realms.mdx

Fixing some typos

jthiller committed:
Linting

deasydoesit committed:
Migration guide updates for mainnet (#1226)

* Updates to exchange operator page

* Update blockchain node migration guide

* Updating based on JH feedback

openxzx committed:
Update default value to GW_API (#1219)

ChewingGlass committed:
Fix config (#1229)

waveform06 committed:
Update vehnt.mdx

Clarifications about veHNT value and the 6%

jthiller committed:
typos and clarification

jamiew committed:
Ledger migration requires Chrome/Brave/HID support (#1240)

Once again the people ignore the best browser of all time, Firefox

jthiller committed:
Explains low balance error.

Commit:
update iot premie docs (#1242)

* update iot premie docs

correction on active hotspots. this does not include data only hotspots

* data only premine note

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

ChewingGlass committed:
Add HST management widget (#1178)

* Add HST management widgget

* More docs

* missing .

* Update docs/solana/hst.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* Update docs/solana/hst.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* Fix dist

* Fix some ledger miration bugs

* Add colleciton key

* Change HST to our fanout impl

* Use real hst

* f

* Fix build

* Fix build

* dep

* fix build

* f

* does this fix

* try client only

* Rename to HST holder

---------

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

jthiller committed:
Add landrush + lockup graphs

jamiew committed:
It's explorer-old not old-explorer

Might be worth setting up the other DNS record too

DeflateAwning committed:
Add spreading factor summary to frequency-plans page

ChewingGlass committed:
Add HNT to fee sol (#1244)

* Add HNT to fee sol

* Fix sidebar

* frontmatter, sidebar, and quick styling

* Small edits to copy

* Use separate svc

* Fixes double inclusion in sidebar.

* Try signAndSend

* Fix est

* Axios get

* decimal error

* Fix

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>
Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

riobah committed:
Fixing the category for iot_reward_shares files (#1247)

waveform06 committed:
Update sol.mdx (#1260)

added line to say use the wallet app browser to open this page to get SOL in the wallet app.

Talitrus committed:
update SOL fee required for lockup (#1261)

* update SOL fee required for lockup

* lien wrap applied

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

deasydoesit committed:
Remove or update occurrences of 'on the Helium L1' (#1259)

* Remove or update occurrences of 'on the helium l1'

* Update docs/oracles/iot-proof-of-coverage-oracles.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* Update docs/oracles/iot-proof-of-coverage-oracles.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* applied line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

waveform06 committed:
Update helium-network.mdx (#1237)

* Update helium-network.mdx

updated with a questions shuffle and replacing HNT with IOT and removal of "Helium Blockchain"

* feedback from suggestions, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update community-governance.mdx (#1236)

* Update community-governance.mdx

Minor text updates

* feedback from suggestions, line wrap, alphabetize lists

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update helium-hotspot-app.mdx (#1231)

* Update helium-hotspot-app.mdx

Changed from Approved Manufacturers to Manufacturers sell approved hotspots

* Update docs/mine-hnt/helium-hotspot-app.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* link cleanup

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

waveform06 committed:
Update twelve-words.mdx (#1234)

* Update twelve-words.mdx

updated with wallet use depreciated text

* consolidating 12/24-word docs to “seed phrase” docs

link fixes
netlify redirects
page cleanup
“current phase” phrasing
sidebar updates

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update ledger.mdx (#1265)

Ledger firmware to be at least 2.1.0 requirement added

samgutentag committed:
hnt-sol swap video demo

jthiller committed:
Add Vote Multiplier to Realms doc. (#1262)

* Adds section on vote multiplier, plus other clarifications.

* Fix sidebar order for sol token

waveform06 committed:
Update maker-apps.mdx (#1233)

* Update maker-apps.mdx

several minor updates

* integrated feedback

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update helium-wallet-app.mdx (#1266)

quick temp update. Removed Solana preview content and updated ledger part and links

waveform06 committed:
Update vehnt.mdx (#1264)

* Update vehnt.mdx

minor text clarifications

* linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Update docs/helium-tokens/sol.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

samgutentag committed:
note to close ledger live before starting migration (#1230)

* note to close ledger live before starting migration

* Update docs/solana/migration/ledger.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

Ryan-Goldstein committed:
Switch MNTD/RAK App Links to Hotspotty Connect (#1272)

* Switch MNTD/RAK App Links to Hotspotty Connect

Official announcement from MNTD indicating that the MNTD app is not supported after the Solana migration and will be removed from app stores soon, and that Hotspotty Connect should be used moving forward: https://cdn.discordapp.com/attachments/404106811252408322/1099171159368544346/image.png

* Update maker-apps.mdx

adding an admonition to make sure this doesn't look like a broken link

* typo

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Clean up Price Oracle docs

jthiller committed:
Update links.

mbthiery committed:
Update ledger migration copy (#1277)

ke6jjj committed:
Add node about compressed NFT RPC requirement to use example.

jthiller committed:
Noting Helius as an option.

DanielChrobak committed:
Add a Solana Migration FAQ page to the docs (#1276)

* Transaction fee correction

Changed to match the now updated and corrected fee shown at https://docs.helium.com/helium-tokens/sol-token

* Create faq.mdx

* Delete faq.mdx

* Create migration

* Update and rename migration to migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update realms.mdx

* Rename migration.mdx to solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

samgutentag committed:
remove $ to remove LaTex formatting, linewraps

waveform06 committed:
Update validators-and-voting-power.mdx (#1271)

* Update validators-and-voting-power.mdx

Updated and removed all references to validators.
Next pass to focus on voting and leave staking delegation to another doc
Can this be renamed to voting power.mdx but also have an alias of the previous name

* line wraps

* move page and create redirects

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update realms.mdx (#1269)

* Update realms.mdx

Some additional clarifications

* rewording from feedback, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
removing cross link prefixes, line wraps (#1282)

samgutentag committed:
redirect (#1284)

jthiller committed:
Clarify delegations and epoch timing (#1286)

* Clarify delegations.

* Consistently capitalize 'landrush'

* Remove redundant 3x

samgutentag committed:
Removing Validator Pages (#1285)

* removing validator pages, images and sidebar sections

* sidebar cleanup

* removing cross links

* redirect for all docs goes to migration guide

* cutting out validator references

* more validator references removed

* restore files from feedback

* feedback revisions

* update explorer to explorer-old domain

* updating more explorer links

* move image file

ChristinaR04 committed:
Update maker-apps.mdx (#1289)

* Update maker-apps.mdx

Edits to include missing info and remove makers that are not MCC approved.

* Update maker-apps.mdx

* small cleanup

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

Sophi committed:
Update compliance-commitee.mdx (#1248)

* Update compliance-commitee.mdx

Updated links and dates

* Update compliance-commitee.mdx

fixed broken link

* linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Decaying vs Cliff

jthiller committed:
Stake Unlocking

jthiller committed:
Copy edits

waveform06 committed:
Update transfer-hotspot.mdx (#1274)

* Update transfer-hotspot.mdx

direct to use Wallet App for hotspot transfers

* changes from feedback

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Add Migration widget to Wallet User page (#1294)

* add widget

* no dos

samgutentag committed:
Solana FAQ Editorial added to Sidebar (#1291)

* rename fies

* merge updates and sidebar fixes

* reorg sectinons

jthiller committed:
Realms Voting Guide (#1295)

* Realms Voting

* Add 'Sign In' explainer to Realms Staking guide

* image optimizations

* section spacers

* key terms to list for formatting

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Ledger Video Migration Demo Video Embed (#1297)

* embed video walk through and fixup admonition

* squircle corners

jthiller committed:
colorize titleBar (#1298)

Jordan-Nguy committed:
Update realms.mdx

Spelling and grammar fixes

Jordan-Nguy committed:
Update vehnt.mdx

Spelling fixes

Jordan-Nguy committed:
Update voting-power.mdx

Spelling and grammar fixes

jthiller committed:
Linting

jthiller committed:
Linting

jadeallenx committed:
Update realms.mdx

Note that staked tokens which are in an active vote cannot be split until the vote is concluded.

jthiller committed:
Update docs/vote-escrow/realms.mdx

waveform06 committed:
Update helium-wallet-app.mdx (#1300)

Added Tokens supported list

jthiller committed:
Fix my bad formatting. (#1304)

lytzV committed:
Create 2023-04-28-offchain-data-transfer-postmortem (#1299)

* Create 2023-04-28-offchain-data-transfer-postmortem

* added author and update frontmatter, linewraps

* file rename with extension

* date formatting fixes from feedback

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
updated docs contribution instructions

Sophi committed:
Update transaction-fees.mdx (#1296)

* Update transaction-fees.mdx

removed the word "full" from Hotspot

* Update docs/blockchain/transaction-fees.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* copy edits and formatting

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

waveform06 committed:
Update voting-power.mdx (#1308)

* Update voting-power.mdx

Put landrush in past tense, clarified positon closure process.

* line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update realms.mdx (#1309)

* Update realms.mdx

Updated closing a position process.

* line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update maker-apps.mdx (#1310)

* Update maker-apps.mdx

Updated LongAP. Location asserts can use the Helium Hotspot App

* trim tail white spaces

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update voting-power.mdx (#1311)

* Update voting-power.mdx

Changed tense about Landrush from future to past.

* line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
taking a pass at cleaning up language (#1314)

matthewcarlreetz committed:
Fix test maker onboarding auth token. (#1316)

waveform06 committed:
Update maker-approval-auditing.mdx (#1232)

* Update maker-approval-auditing.mdx

Added spelling mistake correction and Note about endorsement.

* formatting and copy edits, address feedback

* copy edit

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

Sophi committed:
Update light-hotspots.mdx (#1249)

* Update maker-ethics.mdx

Note about dual mining

* formatting fixup

* Update light-hotspots.mdx

draft

* Update light-hotspots.mdx

* Update light-hotspots.mdx

Updated doc for post HIP-70

* Update light-hotspots.mdx

Removed "Light" term and added an explanation on the bottom

* Update light-hotspots.mdx

* typo

* image updates and language rewrites

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>
Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

ChristinaR04 committed:
Update hotspot-manufacturers.mdx (#1313)

* Update hotspot-manufacturers.mdx

Update to include all approved makers

* page consolidation and rewrites to maker info

* build errors and page redirects

* OG Hotspot support pages marked draft

pages still exist in repo, but wont be published

* more old pages marked as drafts

* removing cross links

* cleanup

* removing outdated docs

* update troubleshooting page

* cleanup

* page shuffling

* frontmatter fixups and sidebar adjustments

* formatting fixes

* added redirects

* sidebar cleanup

* reference link fixes

* redirect for maker apps

* merge conflicts

* merge conflicts

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Add suggested extensions (+ configurations) (#1321)

DeflateAwning committed:
Fix typos

DeflateAwning committed:
Improve API docs post-SOL

jthiller committed:
Testdriving plugins. Misc Cleanup (#1324)

* Typos, grammar, and wording clarification.

* Define style via plugin config

* prettier linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
duplicate page removal and redirect fixups (#1319)

* duplicate page removal

* redirects for broken slugs

jthiller committed:
Establish Updated DC page (#1315)

* Establish DC page

* Remove legacy Data Credit page.

* Fix page crash on clearing input

* Add suggested extensions (+ configurations) (#1321)

* Fix typos

* Improve API docs post-SOL

* Testdriving plugins. Misc Cleanup (#1324)

* Typos, grammar, and wording clarification.

* Define style via plugin config

* prettier linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

* duplicate page removal and redirect fixups (#1319)

* duplicate page removal

* redirects for broken slugs

* Remove legacy Data Credit page.

---------

Co-authored-by: DeflateAwning <11021263+DeflateAwning@users.noreply.github.com>
Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
DC Calculator Updates (#1325)

* On mobile use number pad for inputs.

* handle large numbers on mobile

ke6jjj committed:
Fix reward claiming example.

The process of claiming rewards requires at least one call to the
distributor oracle to ask it to countersign the rewards transaction.
In doing so, the oracle will fill in the "recent block hash" for the
transaction and provide its signature over the same. This means
that the client code which uses this transaction must leave the
recent block hash alone when signing and submitting the transaction
to the chain.

`provider.sendAndConfirm()`, however, implicitly resets the recent
block hash and thus invalidates the transaction.

The fix here is to use a different submission method that leaves
the recent block hash as is.

samgutentag committed:
linewraps

samgutentag committed:
Create slack-notify.yml (#1328)

samgutentag committed:
line wrap for bot test

leogaggl committed:
Revert frequency plan Australia AU915 (#1332)

* Revert frequency pllan Australia

* Remove misleading and incorrect paragraph.

* feedback from review

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
added maker app redirects (#1337)

samgutentag committed:
typo in sub bands (#1338)

Sophi committed:
Update full-hotspots.mdx (#1335)

* Update full-hotspots.mdx

Added section for radio certifications

* Update docs/blockchain/transaction-fees.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* copy edits and formatting

* link fixup

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

capjbadger007 committed:
Update region-plans.mdx (#1343)

* Update region-plans.mdx

Major plan frequencies update.

* Update region-plans.mdx

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
edit hip in browser walkthrough (#1336)

* edit hip in browser walkthrough

* url fix

* Apply suggestions from code review

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* copy edits from feedback

* swap gifs for images

* remove extra admonitions

* community feedback no branching guide

* updated images

* typo

* title update and image optimizations

* figure spacing

* figure spacing

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

samgutentag committed:
Governance Updates (#1334)

* update governance section and links

* cleanup

* clarity and other edits

* update hip section

* url fixup

---------

Co-authored-by: vincenzospaghetti <93289575+vincenzospaghetti@users.noreply.github.com>

samgutentag committed:
Merge branch 'migration/blockchain-api-migration' of https://github.com/helium/docs into migration/blockchain-api-migration

samgutentag commented:
why is it when i run `> git rebase master` i get this giant merge blob of death? no idea

solution: extracted important bits in a few PR #1346
